src/Kernel/Slot.class.st

```diff
@@ -55,6 +55,11 @@ Slot class >> checkValidName: aSymbol [
 
 ]
 
+{ #category : #testing }
+Slot class >> isPseudovariableName: aSymbol [
```

Comment:
For future:
I think we should use the word "reserved". It has better meaning than pseudo or special.
And we can move related methods to ReservedVariable which we have now.

Reply:
ok, added to the backlog (but this has true and false, too... so some rethinking needed anyway, if we remove that we can query ReservedVariable subclases instead of hardcoding)